distributed/dashboard/components/nvml.py

```diff
@@ -181,14 +189,3 @@ def gpu_utilization_doc(scheduler, extra, doc):
     add_periodic_callback(doc, gpu_load, 100)
     doc.add_root(gpu_load.utilization_figure)
     doc.theme = BOKEH_THEME
-
-
-try:
-    import pynvml
-
-    pynvml.nvmlInit()
-except Exception:
-    pass
-else:
-    applications["/individual-gpu-memory"] = gpu_memory_doc
-    applications["/individual-gpu-utilization"] = gpu_utilization_doc
```

Comment:
Did these last two lines go away?  We need them in order to get access to these plots in the JLab extension

Reply:
They are over here now

https://github.com/dask/distributed/pull/3138/files#diff-bb0ae02b02c7f964b31a044e6dc83b19R401-R409

Reply:
Ah, no I see now, they're in scheduler.py